src/components/TeacherComponents/OnboardingForm/getQueryParamsFromOnboardingFormData.ts

```diff
@@ -1,34 +0,0 @@
-import { ParsedUrlQuery, encode } from "querystring";
-
-import {
-  ManualSchoolFormValues,
-  SchoolSelectFormProps,
-  UkSchoolFormValues,
-} from "./OnboardingForm.schema";
-
-export const getQueryParamsFromOnboardingFormData = (
```

Comment:
This was sort of doing what I needed to allow tracking of the accumulated form state through the onboarding flow. 

I've refactored it into `encodeOnboardingDataQueryParam` and `decodeOnboardingDataQueryParam` which collects the form state in base64 JSON making it easy to deserialise back into `OnboardingFormProps`